packages/grafbase-urql-exchange/README.md

```diff
@@ -0,0 +1,87 @@
+# @grafbase/urql-exchange
+
+URQL-exchange for handling Server Sent Events (SSE)
+
+## Getting Started
+
+Follow these steps in a new or existing React application
+
+1. Install the dependencies
+
+```
+npm install urql graphql @grafbase/urql-exchange
+```
+
+2. Create URQL client
+
+```ts
+// client.ts
+import { cacheExchange, createClient, dedupExchange, fetchExchange } from 'urql'
+
+export const client = createClient({
+  url: process.env.GRAFBASE_API_URL,
+  fetchOptions: {
+    headers: { 'x-api-key': process.env.GRAFBASE_API_KEY }
```

Comment:
Frontend examples should use Authorizatiom and not API keys.

Reply:
fixed